src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,108 @@
+import { ChangeEvent, FC } from 'react';
+import cn from 'classnames';
+
+import { SEX_OPTIONS, CENTURIES } from '../../Constants';
+import { SearchLink } from '../SearcLink';
+
+type Props = {
+  activeSex: string | null;
```

Comment:
can we do something like this?
```suggestion
  activeSex: YourEnum;
```